### PR TITLE
Minor fixes to Window Controls Overlay API

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

add `WindowControlsOverlayGeometryChangeEvent` to api group data

specific the spec-urls member of api landing page

add `WindowControlsOverlayGeometryChangeEvent` and `Navigator.windowControlsOverlay` to api landing page

mark the `visible` parameter of `WindowControlsOverlayGeometryChangeEvent` constructor as optional

some other update for pretty style

### Additional details

see https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API and https://wicg.github.io/window-controls-overlay/

### CODE REVIEW

**Comment by skyclouds2001** [comment]
note that the bcd data does not mark `Navigator.windowControlsOverlay` as experimental, while `WindowControlsOverlay` and `WindowControlsOverlayGeometryChangeEvent` does not

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ browser-compat: api.WindowControlsOverlayGeometryChangeEvent.WindowControlsOverl
 
 {{APIRef("Window Controls Overlay API")}}{{SeeCompatTable}}
 
-The **`WindowControlsOverlayGeometryChangeEvent()`** constructor returns a newly created
-{{domxref("WindowControlsOverlayGeometryChangeEvent")}}, representing the current geometry of a desktop Progressive Web App's title bar area.
+The **`WindowControlsOverlayGeometryChangeEvent()`** constructor returns a newly created {{domxref("WindowControlsOverlayGeometryChangeEvent")}}, representing the current geometry of a desktop Progressive Web App's title bar area.
```
Comment (teoli2003) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md:13]: NIT: _newly created_ really means _new_ that is easier to understand.
```suggestion
The **`WindowControlsOverlayGeometryChangeEvent()`** constructor returns a new {{domxref("WindowControlsOverlayGeometryChangeEvent")}} object, representing the current geometry of a desktop Progressive Web App's title bar area.
```

File: files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md [hunk 2]
```diff
@@ -28,8 +27,8 @@ _The `WindowControlsOverlayGeometryChangeEvent()` constructor also inherits argu
   - : A string indicating the event type. It is case-sensitive and browsers set it to `geometrychange`.
 - `options`
   - : An object with the following properties:
-    - `visible`
-      - : A boolean flag that's true when the `titlebarAreaRect` object's values are not 0.
+    - `visible` {{optional_inline}}
+      - : A boolean flag that's true when the `titlebarAreaRect` object's values are not 0, defaults to `false`.
```
Comment (teoli2003) [inline files/en-us/web/api/windowcontrolsoverlaygeometrychangeevent/windowcontrolsoverlaygeometrychangeevent/index.md:31]:
```suggestion
      - : A boolean flag that's true when the `titlebarAreaRect` object's values are not 0. Its default value is `false`.
```

**Review by teoli2003 — APPROVED** [review]
Two nits that I will commit. LGTM:

**Pull request merged**